pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/pubsub/UpdateWorkflowStatusTest.java

```diff
@@ -61,25 +61,14 @@ private static void createDataForParticipant() {
 
     private static void createParticipantData() {
         participantData =
-                new ParticipantData.Builder()
-                        .withDdpParticipantId(participantId)
-                        .withDdpInstanceId(ddpInstanceDto.getDdpInstanceId())
-                        .withFieldTypeId(UPDATE_WORKFLOW_TEST)
-                        .withData(gson.toJson(testParticipantData))
-                        .withLastChanged(System.currentTimeMillis())
-                        .withChangedBy(userDto.getEmail().orElse(""))
-                        .build();
```

Comment:
The same here, builder pattern has its own style how it should be built like here, to make it much easier for eyes to read, to whom should we consult to change that in checkstyle not to touch such construction anymore?